models/phenom_beta_huggingface/config.json

```
@@ -0,0 +1,83 @@
+{
+  "apply_loss_unmasked": false,
+  "architectures": [
+    "MAEModel"
+  ],
+  "crop_size": -1,
+  "decoder": {
+    "_target_": "mae_modules.CAMAEDecoder",
+    "depth": 4,
+    "embed_dim": 256,
+    "mlp_ratio": 4,
+    "norm_layer": {
+      "_partial_": true,
+      "_target_": "torch.nn.LayerNorm",
+      "eps": 1e-06
+    },
+    "num_heads": 16,
+    "num_modalities": 6,
+    "qkv_bias": true,
+    "tokens_per_modality": 256
+  },
+  "encoder": {
+    "_target_": "mae_modules.MAEEncoder",
+    "channel_agnostic": true,
+    "max_in_chans": 11,
+    "vit_backbone": {
+      "_target_": "vit.sincos_positional_encoding_vit",
+      "vit_backbone": {
+        "_target_": "vit.vit_small_patch16_256",
+        "global_pool": "avg"
+      }
+    }
+  },
+  "fourier_loss": {
+    "_target_": "loss.FourierLoss",
+    "num_multimodal_modalities": 6
+  },
+  "fourier_loss_weight": 0.0,
+  "input_norm": {
+    "_args_": [
+      {
+        "_target_": "normalizer.Normalizer"
+      },
+      {
+        "_target_": "torch.nn.LazyInstanceNorm2d",
+        "affine": false,
+        "track_running_stats": false
```

@Laksh47  can we please change this to the following (not sure if you can do null in JSON - but we need to pass `num_features=None` somehow here:
```suggestion
        "_target_": "torch.nn.InstanceNorm2d",
        "num_features": null,
        "affine": false,
        "track_running_stats": false
```